vmdb/app/models/service_template_orchestration.rb

```diff
@@ -4,7 +4,8 @@ def orchestration_template
   end
 
   def orchestration_template=(template)
-    orchestration_templates.clear << template
+    orchestration_templates.clear
+    orchestration_templates << template if template
```

Comment:
This seems like it would be expensive especially on refresh because it would always update...can you verify that refresh doesn't thrash?